filesystem/ios/Plugin/Filesystem.swift

```diff
@@ -136,32 +136,30 @@ import Foundation
     /**
      * Get the SearchPathDirectory corresponding to the JS string
      */
-    @objc public func getDirectory(directory: String) -> FileManager.SearchPathDirectory {
+    public func getDirectory(directory: String?) -> FileManager.SearchPathDirectory? {
         switch directory {
         case "DOCUMENTS":
             return .documentDirectory
         case "CACHE":
             return .cachesDirectory
         default:
```

Comment:
Need to also return documents directory for all other options, should only return nil when no directory is passed